examples/tf/her_ddpg_fetchreach.py

```diff
@@ -51,14 +51,14 @@ def run_task(*_):
             reward_fun=env.compute_reward)
 
         ddpg = DDPG(
-            env,
+            env_spec=env.spec,
             policy=policy,
             policy_lr=1e-3,
             qf_lr=1e-3,
             qf=qf,
             replay_buffer=replay_buffer,
-            plot=False,
             target_update_tau=0.05,
+            n_epoch_cycles=20,
             max_path_length=100,
```

Comment:
I thought this line is deleted in off policy algorithm?

Reply:
Actually not. I keep `n_epoch_cycles` so that an algorithm can compute epoch by `iteration/n_epoch_cycles`. This is a little bit awkward but required in DDPG.

Reply:
I don't quite understand. I meant is `max_path_length=100` not necessary here?